src/main/resources/eu/markov/jenkins/plugin/mvnmeta/MavenMetadataParameterDefinition/help-currentArtifactInfoPattern.html

```diff
@@ -0,0 +1,30 @@
+<!--
+  - The MIT License
+  -
+  - Copyright (c) 2012, AKQA, Georgi "Gesh" Markov
```

Comment:
I just copied and pasted the license disclaimers. For the new files, I'll change the copyright information to my name and company. But I won't touch the disclaimers of the files that existed before.